Components/SettingsBar/Search.tsx

```diff
@@ -0,0 +1,15 @@
+const SettingsSearch = () => {
```

Comment:
There is no search functionality here?

Reply:
@lalitmee Sir, I think the full settings pages will be developed from now as the issue was to only design and implement the Settings Component.
Issue: #233